custom/enikshay/private_sector_datamigration/management/commands/update_names.py

```diff
@@ -0,0 +1,55 @@
+from django.core.management import BaseCommand
+
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.users.models import CommCareUser
+from custom.enikshay.private_sector_datamigration.models import UserDetail, Agency
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+
+    def handle(self, domain, **options):
+        locs_matching_type = SQLLocation.active_objects.filter(
+            domain=domain,
+            location_type__code__in=[
+                'pac',
+                'pcp',
+                'plc',
+                'ps-fieldstaff',
```

Comment:
I think this should be `fo`. the `ps-fieldstaff` is the usertype assigned to that location.